runtime/rasdump/javadump.cpp

```diff
@@ -993,6 +994,13 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	_OutputStream.writeCharacters(_VirtualMachine->memoryManagerFunctions->omrgc_get_version(_VirtualMachine->omrVM));
 	_OutputStream.writeCharacters("\n");
 
+#ifdef VENDOR_NAME
+	/* Write the VM version data */
+	_OutputStream.writeCharacters("1CI");
+	_OutputStream.writeCharacters(VENDOR_NAME,3);
+	_OutputStream.writeCharacters("VERSION  " VENDOR_SHA "\n");
```

Comment:
Now that the name and SHA are separate, the whole name should be written; also these three calls to `writeCharacters` can be collapsed.